energy-growers/src/components/SubMenu/SubMenu.scss

```
@@ -3,10 +3,52 @@
 .eg-sub-menu {
   width: 100vw;
   padding-top: rem(31);
-  padding-bottom: rem(211);
+  padding-bottom: rem(52);
+
+  @media screen and (min-width: rem(map-get($breakpoints, sm))) {
+    padding-top: rem(39);
+    padding-bottom: rem(10);
```

```suggestion
    padding: rem(39) 0 rem(10);
```
lo dovrai fare per tutti cosi, cosi abbiamo solo un rigo invece di 2